### DISCUSSION TITLE
Mailserver template is missing

### DISCUSSION THREAD

**Discussion by glebedenko** [Dokploy/dokploy#710] [Q&A]
Hello. I've been waiting for a mail server template for some time and finally it was introduced in [v.0.11.0](https://github.com/Dokploy/dokploy/releases/tag/v0.11.0). 

I've upgraded my Dokploy instance from v0.7.2 to v0.11.2 and found out that there is no actual templates for Mailserver and ERPNext that were added together by @SashaGoncharov19 in #458. I failed to find that commit in canary's commit history. Failed to find any related changes in templates.ts as well. 

I'm confused. Was canary's commit history overwritten?

**Comment by SashaGoncharov19** [comment]
@ObsidianLord Will check, it. Thanks
